### PR TITLE
feat: add mouse.wheel method

### PR DESCRIPTION
**Pull request by jiladahe1997**
add the mouseWheel event method . This method could scroll the page to a specific coordinate .

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -1589,6 +1590,13 @@ Dispatches a `mousemove` event.
 
 Dispatches a `mouseup` event.
 
+#### mouse.wheel(deltaX, deltaY)
+- `deltaX` <[number]>
+- `deltaY` <[number]>
+- returns: <[Promise]>
+
+Dispatches a `mousewheel` event. Please note that it will take a very short time to scroll the page, so maybe you need use Page.waitFor(). The method Page.screenshot will scroll the page to 0, 0.
```
Comment (vsemozhetbyt) [inline docs/api.md:1598]: Not sure, but maybe:
`` Page.waitFor() `` -> `` `page.waitFor()` ``
`` Page.screenshot `` -> `` `page.screenshot()` ``

**Comment by aslushnikov** [comment]
@JoelEinbinder can you please take a look?

**Review by JoelEinbinder — COMMENTED** [review]
I don’t think we should add this until we properly wait for the scrolling on the back end

**Comment by aslushnikov** [comment]
@JoelEinbinder do you have an upstream bug we can refer to here?

**Comment by aslushnikov** [comment]
DevTools protocol method for dispatching mouse wheel events is broken now: it needs to wait for the scrolling to finish before returning.

This has to be fixed upstream.

**Pull request closed without merging**